modules/samples/producers/websocket/src/main/java/org/wso2/carbon/sample/websocket/Websocket.java

```diff
@@ -48,6 +48,9 @@ public static void main(String args[]) {
         String filePath = args[2];
         String sampleNumber = args[3];
 
+        //for host
+        String host=args[4];
+
         if (url.isEmpty()) {
             System.out.println("Starting Websocket publisher on Server Mode");
```

Comment:
@sacjaya 
Thanks for the feedback. Fixed in PR: https://github.com/wso2/product-cep/pull/188